models/task.go

```diff
@@ -22,6 +26,24 @@ type TaskLinks struct {
 	Job  string `json:"job"`
 }
 
+// TaskName - iota enum of possible task names
+type TaskName int
+
+// ParseTaskName returns a TaskName from its string representation
+func ParseTaskName(taskNameStr string) (TaskName, error) {
+	cfg, err := config.Get()
+	if err != nil {
+		return -1, fmt.Errorf("%s: %w", errors.New("unable to retrieve service configuration"), err)
+	}
+	taskNameValues := strings.Split(cfg.TaskNameValues, ",")
```

Comment:
This can be done once on startup, instead of calculating this logic everytime a request comes in - this will increase performance

Also it would be better to create a map instead of an array, so we can just lookup the taskname in map instead of using a for loop when the request comes in - again increases performance.